QueenSolutionConsole.cs

```diff
@@ -124,6 +117,47 @@ private static int[] YFlip(int[] placed)
 
             return test;
         }
+        private static int[] Rotate90(int[] placed)
```

Comment:
This should also have a comment "not used yet".

Reply:
good point.  I'll update.